spec/index.adoc

```diff
@@ -4109,14 +4126,15 @@ callbacks = λ(callbacks, response, refunded_cycles, sysenv, available) → λ w
     if typeof(func) ≠ func (i32) -> () then Trap
 
     let es = ref { empty_execution_state with
-      wasm_state;
+      wasm_state = wasm_state;
     }
     func<es>(callbacks.on_cleanup.env)
     Return {
       new_state = es.wasm_state;
       new_calls = [];
       response = NoResponse;
       cycles_accepted = 0;
+      cycles_used = 0;
```

Comment:
Is this the right choice?

Reply:
Do I understand correctly that this would mean: If the cleanup callback is run (and completes successfully), no cycles will be charged? In that case, it doesn't seem to be what I would expect. @bogwar ?

Reply:
Your understanding is right. If we want to charge cycles, then I can just replace `0` by `es.cycles_used`. I was just not sure what the choice should be.

Reply:
We do charge cycles for executing the cleanup callback

Reply:
I've just fixed that, but still the cycles consumed by the actual callback (in the try branch before `Trap`) are not considered.

Reply:
They're now considered.